templates/build/small-card-detail.build.hbs

```diff
@@ -54,12 +54,21 @@
         \{{#ifCond type '==' 'image'}}
           \{{#if content}}
             \{{#if labelEnabled}}
-            <div class="small-card-list-detail-label">\{{label}}</div>
+	            <div class="small-card-list-detail-label">\{{label}}</div>
```

Comment:
![image](https://user-images.githubusercontent.com/53430352/113684907-5d704400-96ce-11eb-91d9-bf44fd71a94c.png)
About this indent. When I edit it in the VSCode indent is okay, and I don't know why in the git it displays wrong.

Reply:
@YaroslavOvdii There's probably a tab character in there somewhere.

Reply:
@tonytlwu Yes, you were right about that. Removed tab character